src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -346,6 +347,12 @@ private static void syncConfig(boolean load)
         alwaysSetupTerrainOffThread = prop.getBoolean(false);
         prop.setLanguageKey("forge.configgui.alwaysSetupTerrainOffThread");
         propOrder.add(prop.getName());
+        
+        prop = config.get(Configuration.CATEGORY_CLIENT, "allowEmissiveItems", true,
```

Comment:
Maybe make a function in MinecraftForgeClient (something like enableEmissiveItems) so mods that use that feature can enable it themselves and do not rely on the user doing it. This also avoids the performance overhead when no mod is using the feature

Reply:
This needs rebenchmarking, preferably after addressing the optimisation issues. Ideally there shoudn't be any noticable performance overhead if nothing is using the feature, which would make that unnecessary - it'd be mostly about choosing whether to spend performance on actually having the effects.